python/cairo-addons/src/cairo_addons/testing/runner.py

```diff
@@ -325,7 +339,8 @@ def _run(entrypoint, *args, **kwargs):
         #   and performs security checks
         # ============================================================================
         runner.end_run(disable_trace_padding=False)
-        cumulative_retdata_offsets = serde.get_offsets(return_data_types)
+        cairo_types = [item["type"] for item in return_data_types]
```

Comment:
why do we need this? is it a boyscout commit?

Reply:
skip returning blake2s_ptr